plugins/dlab_templates/dlab_templates/infrastructure/views.py

```diff
@@ -0,0 +1,9 @@
+from flask_restful import Resource
```

Comment:
this is not a view it is part of a routing as we discuss

Reply:
discussed